tests/ImageSharp.Tests/Formats/Pbm/PbmDecoderTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.IO;
+using SixLabors.ImageSharp.Formats.Pbm;
+
+using Xunit;
+using static SixLabors.ImageSharp.Tests.TestImages.Pbm;
+
+// ReSharper disable InconsistentNaming
+namespace SixLabors.ImageSharp.Tests.Formats.Pbm
+{
+    [Trait("Format", "Pbm")]
+    public class PbmDecoderTests
+    {
+        [Theory]
+        [InlineData(BlackAndWhitePlain, PbmColorType.BlackAndWhite)]
+        [InlineData(BlackAndWhiteBinary, PbmColorType.BlackAndWhite)]
+        [InlineData(GrayscalePlain, PbmColorType.Grayscale)]
+        [InlineData(GrayscaleBinary, PbmColorType.Grayscale)]
+        [InlineData(GrayscaleBinaryWide, PbmColorType.Grayscale)]
+        [InlineData(RgbPlain, PbmColorType.Rgb)]
+        [InlineData(RgbBinary, PbmColorType.Rgb)]
+        public void PpmDecoder_CanDecode(string imagePath, PbmColorType expectedColorType)
```

Comment:
Any chance you can add pixel data validation to these tests?
Instead of relying on an ImageMagick reference decoder, you can push the expected output as a PNG in the `ReferenceOutput` folder, and do something similar `JpegDecoderTests` do:
- [Tests]( https://github.com/SixLabors/ImageSharp/blob/d81c511f068ac0fdb580310fe0d2328ca713e919/tests/ImageSharp.Tests/Formats/Jpg/JpegDecoderTests.Baseline.cs)
- [Expected output](https://github.com/SixLabors/ImageSharp/tree/master/tests/Images/External/ReferenceOutput/JpegDecoderTests)

If you are sure your current implementation is correct, you can just copy the output of [`DebugSave`](https://github.com/SixLabors/ImageSharp/blob/d81c511f068ac0fdb580310fe0d2328ca713e919/tests/ImageSharp.Tests/TestUtilities/TestImageExtensions.cs#L29) there without bothering to convert images with an external app.

We also need at least one test to validate `Image.Load<T>` works with multiple pixel types.

Reply:
Sure, see PbmDecoderTests for this addition.

Also, I wrote a more eleborate description of the image formats in the code documentation of the PbmEncoder and PbmDecoder. Including a link to the specification.

Reply:
Sure, see PbmDecoderTests for this addition.

Also, I wrote a more eleborate description of the image formats in the code documentation of the PbmEncoder and PbmDecoder. Including a link to the specification.